src/js/controller/captions.js

```diff
@@ -26,7 +26,7 @@ const Captions = function(_model) {
     }, this);
 
     // Update tracks once we know "renderCaptionsNatively" based on provider
-    _model.on('itemReady', () => {
+    _model.on('change:itemReady', () => {
```

Comment:
This will fire when it's changed to false and to true. It should only run when true. You can prevent change:itemReady with false if you set false directly on the `model.attributes`.